app/assets/javascripts/Patient.js

```diff
@@ -0,0 +1,169 @@
+var Schema = mongoose.Schema;
+var Code = Schema.Types.Code;
+var Interval = Schema.Types.Interval;
+var Quantity = Schema.Types.Quantity;
+var Integer = Schema.Types.Integer;
+var Array = Schema.Types.Array;
+var String = Schema.Types.String;
+var Float = Schema.Types.Float;
+var Time = Schema.Types.Time;
+var Datatype = require('./basetypes/Datatype');
+
+var PatientSchema = Datatype.extendSchema(Datatype.DatatypeSchema,
+  birth_datetime: DateTime,
+  qdm_version: { type: String, default: "5.3" },
+  ethnicity: Code,
+  race: Code,
+  sex: Code,
+  given_names: [String],
+  family_name: String,
+  bundle_id: String,
+  // These are the "data criteria", or QDM datatype elements that exist on a
+  // patient.
+  data_elements: []
+});
+
+// Returns an array of history elements that exist on this patient, that
```

Comment:
You changed the `history_elements` terminology to `data_elements` in the code itself--the comments should be changed now too.

Reply:
Good catch